manifests/01-openshift-imagestream.yaml

```diff
@@ -0,0 +1,23 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  namespace: openshift
+  name: driver-toolkit
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  tags:
+  - name: latest
+    importPolicy:
+      scheduled: true
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/driver-toolkit:v4.0
+  - name: 0.0.1-snapshot-machine-os 
+    importPolicy:
+      scheduled: true
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/driver-toolkit:v4.0
```

Comment:
To make it more obvious that this is just a placeholder for the `image-references` replacement, I'd consider using `example.com/image-reference-placeholder:driver-toolkit` or some such, that folks won't mistake as a real pullspec.

Reply:
+1, I was going to ask what this `:v4.0` was referring to ;-)